src/protocols/gwmp/semtech_udp.erl

```diff
@@ -83,44 +79,6 @@ pull_data(Token, MAC) ->
 pull_ack(Token) ->
     <<?PROTOCOL_2:8/integer-unsigned, Token:2/binary, ?PULL_ACK:8/integer-unsigned>>.
 
-%%%-------------------------------------------------------------------
```

Comment:
Any reason for this to be removed?

Reply:
This is no longer used.
We are doing it a different way.
cc/ @michaeldjeffrey

Reply:
When this was originally done there were a few things in different places.
- frequencies were sent to us as floats.
- we had a special version of `jsx` that allowed for float formatting.

`jsx` would give us a value, and we'd give it back the string to insert into it's binary. 
We're not using that version of `jsx` so any special treatment of values before they get decoded is mostly useless.

However, now we're getting frequencies in Hertz, so we don't have to worry about a frequency coming in with 12+ decimal places. 
`jsx` seems to be consistent serializing floats that we've pre-truncated to 4 decimal places.

---
We added the decimal truncation to anywhere that uses `hpr_packet_up:frequency_mhz/1` and that's about as much as we can do without getting into the serializer game again.
And not running through this function means things like `lat` and `lon` won't be truncated.